### ISSUE TITLE
Python 3: NVDA cannot exit itself properly

### ISSUE THREAD

**Issue by zstanecic** [nvaccess/nvda#9932]
### Steps to reproduce:
1. Press insert+q
2. select to exit NVDA and press enter.
### Actual behavior:
NVDA still works and speaks, nVDA system tray icon is exited, along with the NVDA menu and pressing NVDA+q fires up an error sound and logs cannot be optained
### Expected behavior:
NVDA should exit normally and clear
#### NVDA installed/portable/running from source:
portable
#### NVDA version:
latest treshould based on py3, compiled at 12 utc
#### Windows version:
windows 10, 1903
#### Name and version of other software in use when reproducing the issue:
not applicable, even with addons disabled the issue is the same
#### Other information about your system:
not applicable
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no, and not applicable, as this problem started to occur in py3

**Comment by josephsl** [comment]
Hi, we need an old log please. Thanks.

**Comment by zstanecic** [comment]
Say me, how i can grab a log from the copy, which is stuck...

**Comment by josephsl** [comment]
Hi, one way is restarting NVDA, then opening temp folder and locating nvda-old.log, as it may contain needed info at the end. Thanks.

**Comment by zstanecic** [comment]
What is more strange, i cannot reproduce it always, and it is reproducible in the random situations

**Comment by zstanecic** [comment]
This problem somehow disappeared, so i will close it for now, if i can do it, as i cannot invoke /reproduce it wildly

**Comment by zstanecic** [comment]
I've closed it, as i tried to reproduce it, and it dissappeared in the same way it came
